pyproject.toml

```diff
@@ -75,19 +75,139 @@ style = "pep440"
 coverage = "^6.2"
 requests-cache = "^1.2.0"
 
+# https://docs.astral.sh/ruff/settings/
 [tool.ruff]
+extend-exclude = []
```

Comment:
As far as I see it makes no difference to add this "extend-exclude" line so you could leave it out.

Reply:
To avoid a load of rebasing (and rerunning of tests), I will remove this line in a future PR.